상세페이지-1

Yunjigong opened the pull request:
AppBar, 노선명, 주요역, 시간표, 노선을 구현했습니다. 시간표는 웹페이지와 다르게 Expansion Tile을 이용하였고 아직 노선에서 색상을 번갈아서 지정하는 것은 못했습니다.

yunji420 committed:
상세페이지-1

AppBar, 노선명, 주요역, 시간표, 노선을 구현하였습니다. 시간표는 웹페이지와 다르게 Expansion Tile을 이용하였고 아직 노선에서 색상을 번갈아서 지정하는 것은 못했습니다.

junwha commented on lib/Detailed-Page.dart:
```diff
@@ -0,0 +1,90 @@
+}
+
+class _DetailedPageState extends State<DetailedPage> {
+  final List<String> _RouteList = ['337(삼남 순환)'];
```
카멜케이스를 사용할 때 변수명의 첫번째는 소문자로 적어주세요!

junwha commented on lib/Detailed-Page.dart:
```diff
@@ -0,0 +1,90 @@
+                    height: 40,
+                    padding: EdgeInsets.all(10.0),
+                    decoration: BoxDecoration(color: Colors.purpleAccent.withOpacity(0.1)),
+                    child: Text('노선명: '+ _RouteList[0]),
```
항상 0에만 접근할 경우 리스트일 필요가 없어요!

junwha commented on lib/Detailed-Page.dart:
```diff
@@ -0,0 +1,90 @@
+                borderRadius: BorderRadius.circular(5.0),
+                border: Border.all(color: Colors.black12),
+              ),
+              child: Text('주요역: '+ _MainStation[0]),
```
여기도 마찬가지입니다
이 데이터는 리스트여야 하기 때문에 각 리스트를 String으로 묶어줘야합니다!

junwha commented on lib/Detailed-Page.dart:
```diff
@@ -0,0 +1,90 @@
+            ExpansionTile(
+              title: const Text('시간표'),
+              children: <Widget>[
+                ListTile(title: Text(_TimeTable[0]),)
```
여기는 각 리스트를 Text widget List로 만들어주면 됩니다.

https://masswhale.tistory.com/entry/Dart%EC%96%B8%EC%96%B4%EA%B3%B5%EB%B6%80-25List-%EC%8B%AC%ED%99%94-forEach-map-reduce-fold

yunji420 committed:
0714 상세페이지

수정사항
1. 변수에 따라서 노선명, 주요역값이 바뀌도록 했습니다.
2. 시간표 list 임의로 만들어서 변수에 따라서 그 노선에 맞는 모든 시간이 나타나도록 했습니다.
3. 노선에서 datarow 부분의 색상을 번갈아서 나타나도록 했습니다.
4. 버스 각 노선의 정류장을 리스트로 받으면 그 정류장들을 data table 에 순서대로 표시하도록 했습니다.

yunji420 committed:
0718 상세페이지

1. 스크롤바 만들었습니다.

ghost commented:
![2022-07-18 (5)](https://user-images.githubusercontent.com/103197388/179449855-2335b81b-323e-4e9f-b5b3-7d68b1e2343c.png)

처음 pull request 했을 때에서 수정한 점은
1. 리뷰해주신 내용 수정
2. 노선에서 색상 번갈아서 지정하고 리스트 값들 입력되도록 수정
3. 스크롤 바 생성

아직 못 한 점은
1. main.dart 파일과 연결
2. 운행중인 버스가 어디 지나고 있는지 표시하는 부분
못했습니다!
